doc/source/transformation_over_time.rst

```diff
@@ -3,41 +3,23 @@ Managing Transformations over Time
 ==================================
 
 In applications, where the transformations between coordinate frames are 
-changing over time, prior to using 
-:class:`~pytransform3d.transform_manager.TransformManager` 
-you should consider interpolating between samples.
-
-We can visualize the problem in the figure below. The tranformation graph with 
-3 coordinate systems: W (world), A, B is visualized over time. Each circle 
-represents a sample (measurement) holding the transformation from the parent 
+dynamic (i.e. changing over time), consider using 
+:class:`~pytransform3d.transform_manager.TemporalTransformManager`. In contrast to
+the :class:`~pytransform3d.transform_manager.TransformManager`, 
+which deals with static transfomations, it provides an
+interface for the logic needed to interpolate between transformation samples 
+available over time.
+
+We can visualize the lifetime of two dynamic transformations 
+(i.e. 3 coordinate systems) in the figure below.
+Each circle represents a sample (measurement) holding the transformation from the parent 
 to the child frame.
 
 .. figure:: _static/tf-trafo-over-time.png
     :width: 60%
     :align: center
 
-Let's assume we want to inspect the situation at the timestep :math:`t_q` (q=query). 
-Further we want to transform points from A to B. As shown in the previous example
-:class:`~pytransform3d.transform_manager.TransformManager` is helpful to deal 
-with transformation graphs.
-
-However, prior to creating the manager, we have to interpolate the transformation streams
-at a timestep of interest. This is visualized with small circles filled with color of
-the according transformation direction.
-
-The function for interpolation from an timeseries of transformation is shown below:
-
-.. literalinclude:: ../../examples/plots/plot_interpolation_for_transform_manager.py
```

Comment:
What do we do about the example? Can we adapt it to use the new features?

Reply:
Yes, sure I will adapt it after we discussed the numpy/pandas input type.

Reply:
> whether to expect/require normalized quaternions

Your decision. It is usually not required to normalize quaternions in pytransform3d's interfaces because `check_quaternion` does this automatically. However, if this increases computational overhead significantly, it might not make sense and we can just document the interface properly.

Reply:
Since the checking is done once during `add_transform` we can deal with that for now ...